model/client.go

```diff
@@ -787,3 +787,60 @@ func (c *Client) DeleteWebhook(webhookID string) error {
 		return errors.Errorf("failed with status code %d", resp.StatusCode)
 	}
 }
+
+// LockAPIForCluster locks API changes for a given cluster.
+func (c *Client) LockAPIForCluster(clusterID string) error {
+	return c.makeSecurityCall("cluster", clusterID, "api", "lock")
```

Comment:
Not a suggestion for right now but it occurs to me that it might behoove us to use constants for stuff like `"cluster"` and `"installation"` here and elsewhere in the repo

Reply:
Yep! Probably worth doing.